src/target/page/TargetPage.tsx

```diff
@@ -9,7 +9,7 @@ const hasText = (str: string | null | undefined): boolean =>
 
 const TargetInfo = ({ target }: { target?: Target }): JSX.Element => (
   <>
-    <h1>{target?.name}</h1>
+    <h1 className="target-info-title">{target?.name}</h1>
```

Comment:
What should we display here is name is missing? Could we add something like 'Nimeämätön kohde'? I know this is little offtopic regarding what this pr is about. This probably is not a priority, and also pretty rare condition and situation if the name would be missing.

Reply:
I can but a default text: "Toimipisteellä ei ole nimeä"